chunks/tool-nix/Dockerfile

```diff
@@ -2,6 +2,7 @@ ARG base
 FROM ${base}
 
 ARG NIX_VERSION
+ARG NIX_CONFIG=""
```

Comment:
You are setting this to empty value and then later using it to write to a file. However, this value is also being supplied from  chunk file. How would this behave?

Reply:
This is to let variants specify optional additional nix config options. Some features in new versions need to be enabled with an experimental flag, but this can cause older versions of nix to show a warning if those flags are defined and unknown.